gocourse08/collartask/animal/animal.go

```diff
@@ -0,0 +1,49 @@
+package animal
+
+type AnimalType string
+
+const (
+	Bear  AnimalType = "bear"
+	Tiger AnimalType = "tiger"
+	Lion  AnimalType = "lion"
+)
+
+//type Animal interface {
+//	Typify() string
+//	RecordBreathing(breathingData float64)
+//	RecordSound(soundData float64)
+//}
+
+type DataAnimal[T float64] struct {
+	AnimalType  string
```

Comment:
`AnimalType` має мати тип `AnimalType` замість `string`:
```suggestion
	AnimalType  AnimalType
```

Так більш типобезпечно.